File: FaceSample/app/src/main/java/com/regula/facesample/items/features/LivenessHintAnimationItem.java
package com.regula.facesample.items.features;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;
import com.regula.facesdk.configuration.LivenessConfiguration;

/**
 * Created by Sergey Yakimchik on 23.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class LivenessHintAnimationItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        LivenessConfiguration configuration = new LivenessConfiguration.Builder()
                .setShowHelpTextAnimation(false)
                .build();
        FaceSDK.Instance().startLiveness(context, configuration, livenessResponse -> {});
    }

    @Override
    public String getTitle() {
        return "Disable Liveness HintView animation";
    }

    @Override
    public String getDescription() {
        return "Disables blinking HintView animation for Liveness.";
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/features/LivenessHintAnimationItem.kt
package com.regula.facesamplekotlin.items.features

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.configuration.LivenessConfiguration
import com.regula.facesdk.model.results.LivenessResponse

class LivenessHintAnimationItem : CategoryItem() {
    override fun onItemSelected(context: Context) {
        val configuration = LivenessConfiguration.Builder()
            .setShowHelpTextAnimation(false)
            .build()
        FaceSDK.Instance().startLiveness(
            context, configuration
        ) { livenessResponse: LivenessResponse? -> }
    }

    override val title: String
        get() = "Disable Liveness NotificationTextView animation"

    override val description: String
        get() = "Disables blinking NotificationTextView animation for Liveness."
}